src/components/tiger/tiger.tsx

```diff
@@ -0,0 +1,252 @@
+    }, []);
+
+    /**
+     * handleProcessTermination: Callback to handle the termination of the child process.
+     * Once the process termination is handled, it clears the process PID reference and
+     * deactivates the loading overlay.
+     * @param {object} param - An object containing information about the process termination.
+     * @param {number} param.code - The exit code of the terminated process.
+     * @param {number} param.signal - The signal code indicating how the process was terminated.
+     */
+    const handleProcessTermination = useCallback(
+        ({ code, signal }: { code: number; signal: number }) => {
+            // If the process was successful, display a success message.
+            if (code === 0) {
+                handleProcessData("\nProcess completed successfully.");
+
+                // If the process was terminated manually, display a termination message.
+            } else if (signal === 15) {
+                handleProcessData("\nProcess was manually terminated.");
+
+                // If the process was terminated with an error, display the exit and signal codes.
+            } else {
+                handleProcessData(`\nProcess terminated with exit code: ${code} and signal code: ${signal}`);
+            }
+
+            // Clear the child process pid reference. There is no longer a valid process running.
+            setPid("");
+
+            // Cancel the loading overlay. The process has completed.
+            setLoading(false);
+        },
+        [handleProcessData] // Dependency on the handleProcessData callback
+    );
+
+    /**
+     * Handles form submission for the Tiger component.
+     * @param {FormValuesType} values - The form values containing the target domain.
+     */
+    const onSubmit = async (values: FormValuesType) => {
+        // Activate loading state to indicate ongoing process
+        setLoading(true);
+
+        // Construct arguments for the Tiger command based on form input
+        let args = [];
+        args = [values.targetIP, "-l", values.auditLevel];
+
+        // Check if reportFile has a value and push it to args
+        if (values.reportFile) {
+            args.push("-R", values.reportFile);
+        }
+
+        // Check if enableModules has a value and push it to args
+        if (values.enableModules) {
+            args.push("-E", values.enableModules);
+        }
+
+        // Check if excludeModules has a value and push it to args
+        if (values.excludeModules) {
+            args.push("-X", values.excludeModules);
+        }
+
+        if (verboseMode) {
+            args.push("-v"); // Add verbose mode option if enabled
+        }
+
+        // Execute the Tiger command via helper method and handle its output or potential errors
+        CommandHelper.runCommandWithPkexec("tiger", args, handleProcessData, handleProcessTermination)
+            .then(() => {
```

Comment:
By entering output and pid into the callback, the command will be displayed in the output. 
```suggestion
            .then(( { output, pid }) => {
```